main.go

```diff
@@ -42,11 +72,44 @@ func main() {
 		}
 		yellow("SERVED", fmt.Sprintf("%s %s", r.URL.Path, cacheDiff))
 
-		if r.URL.Path == compressedWASMFile {
-			w.Header().Set("Vary", "Accept-Encoding")
-			w.Header().Set("Content-Encoding", "br")
-			w.Header().Set("Content-Type", "application/wasm")
```

Comment:
Just a note for myself.
Going to have to add this back otherwise I'm getting an error (at least in firefox)